org.civicrm.bootstrapcivicrm/scss/civicrm/vacancies/_vacancies.scss

```diff
@@ -1,31 +1,169 @@
-.crm-container {
-  .advmultiselect {
-    input[type='button'] {
-      bottom: 21px !important;
-      position: relative;
-    }
-  }
-}
-
-.form-layout-compressed {
-  td {
-    &.label {
-      text-align: left !important;
+.page-civicrm-vacancy {
+  .crm-container {
+    .advmultiselect {
+      input[type='button'] {
+        bottom: 21px !important;
+        position: relative;
+      }
     }
   }
 
   .CRM_HRRecruitment_Form_HRVacancy {
-    .crm-form-block {
-      padding-top: 15px;
+    .crm-block {
+      padding-top: 20px;
 
       > table.form-layout-compressed,
       > fieldset {
-        margin: 15px;
+        margin: 20px;
       }
 
       > table.form-layout-compressed {
         margin-top: 0;
+
+        td {
+          &.label {
+            text-align: left !important;
+          }
+
+          > .select2-container,
+          > .crm-form-text,
+          > .crm-form-date-wrapper .crm-form-text:nth-child(2) {
+            min-width: 200px !important;
+            width: 200px !important;
+
+            &.hasDatePicker {
+              min-width: 169px !important;
+              width: 169px !important;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .crm-block.crm-form-block {
+    .crm-button.crm-button_qf_HRVacancy_next,
+    .crm-profile-selector-preview {
+      @extend %btn-primary;
+    }
+
+    .crm-button-type-cancel {
+      @extend %btn-default;
+    }
+  }
+
+  #addMorePermission {
+    @extend %btn-primary;
+
+    > .add-icon {
+      background-image: none;
+      text-indent: 0;
+      margin-right: 3px;
+
+      &:after {
+        content: '\f067';
+        position: relative;
+        top: -1px;
+        font-family: 'FontAwesome';
+        color: #fff !important;
       }
     }
   }
+
+  .permission-delete-link {
+    border: none !important;
+    padding: 0 !important;
+
+    &:hover,
+    &:focus {
+      background: none !important;
+    }
+
+    > .icon {
+      margin-left: 0;
+      background-image: none !important;
+      text-indent: 0 !important;
+      display: inline !important;
+
+      &:hover,
+      &:focus {
+        background: none !important;
+      }
+
+      &:after {
+        content: '\f014';
+        top: -3px;
+        position: relative;
+        font-family: 'FontAwesome';
+        font-weight: normal;
+        color: $brand-danger !important;
+      }
+    }
+  }
+
+  .crm-option-edit-link.crm-hover-button {
+    border: none !important;
+
+    &:hover,
+    &:focus {
+      background-color: transparent !important;
+    }
+
+    > .crm-i {
+      color: $brand-primary !important;
+    }
+  }
+
+  .crm-submit-buttons {
+    padding: 20px !important;
+    margin: 0 !important;
+    min-height: 35px !important;
+  }
+
+  .CRM_HRRecruitment_Form_HRVacancy {
+    tbody:first-child {
+      tr:nth-child(8),
```

Comment:
As for javascript part, you're targeting a specific type of element only by using the parent's position. Isn't there something in the markup to identify fields for which you need to add the icon?